web/src/main/java/com/futurewei/alcor/web/restclient/MacManagerRestClient.java

```diff
@@ -40,11 +40,12 @@ public void releaseMacAddress(String macAddress) throws Exception {
         restTemplate.delete(url);
     }
 
-    public MacStateJson allocateMacAddress(String projectId, String vpcId, String portId) throws Exception {
+    public MacStateJson allocateMacAddress(String projectId, String vpcId, String portId, String mac) throws Exception {
```

Comment:
is this a macAddress or macId?

Reply:
macAddress

Reply:
Let me rename it.

Reply:
No worries. I made changes to expedite the merge.